src/app/users.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@angular/core';
+
+interface User {
+  id: string;
+  firstName: string;
+  lastName: string;
+  age: number;
+  login: string;
+  password: string;
+  isDeleted: boolean;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersService {  
+
+  private users: User[] = [
```

Comment:
YOu can move this mock data into a `users.mock.ts` file and import it in the service. This will keep the service file short and concise